src/core/memory.d

```diff
@@ -159,6 +159,17 @@ struct GC
     @disable this();
 
     /**
+     * Aggregation of GC stats to be exposed via public API
+     */
+    static struct Stats
+    {
+        /// total size of GC heap
+        size_t heapsize;
+        /// free bytes on the GC heap (might only get updated after a collection)
+        size_t freesize;
```

Comment:
Any reason for the `alllowercase`? Since we are adding something new, we might follow the current style guide, right?